frontend/src/features/projects/common/projectConverter.ts

```diff
@@ -246,7 +242,7 @@ export const toApiProperty = (
       {
         parcelId: isParcelOrSubdivision(property) ? property.id : undefined,
         buildingId: property.propertyTypeId === PropertyTypes.BUILDING ? property.id : undefined,
-        value: property.market,
+        value: property.market || 0,
```

Comment:
Without this it will attempt to return `""` back to be deserialized which will result in a failure in the API.